lnbits/bolt11.py

```diff
@@ -79,7 +80,7 @@ def decode(pr: str) -> Invoice:
         elif tag == "p" and data_length == 52:
             invoice.payment_hash = _trim_to_bytes(tagdata).hex()
         elif tag == "x":
-            invoice.expiry = tagdata.uint
+            invoice.expiry = tagdata.uint  # type: ignore
```

Comment:
- can an `assert tagdata, "..."` solve the `# type: ignore`?

Reply:
same as above, maybe it works with a default value :). i do not trust it for this PR though. i rather go safe, with bolt11.py